pkg/controller/kogitosupportingservice/kogitodataindex_controller.go

```diff
@@ -174,7 +174,10 @@ func (r *ReconcileKogitoDataIndex) Reconcile(request reconcile.Request) (result
 	if resultErr = infrastructure.InjectDataIndexURLIntoKogitoRuntimeServices(r.client, instance.Namespace); resultErr != nil {
 		return
 	}
-	if resultErr = infrastructure.InjectDataIndexURLIntoMgmtConsole(r.client, instance.Namespace); resultErr != nil {
+	if resultErr = infrastructure.InjectDataIndexURLIntoSupportingService(r.client, instance.Namespace, appv1alpha1.MgmtConsole); resultErr != nil {
+		return
+	}
+	if resultErr = infrastructure.InjectDataIndexURLIntoSupportingService(r.client, instance.Namespace, appv1alpha1.TaskConsole); resultErr != nil {
```

Comment:
Maybe we could refactor this function to receive a slice of resources instead?

```go
infrastructure.InjectDataIndexURLIntoSupportingService(r.client, instance.Namespace, appv1alpha1.TaskConsole, appv1alpha1.MgmtConsole)
```

Reply:
done